react/AutocompleteBlock.tsx

```diff
@@ -410,12 +410,15 @@ const AutocompleteBlock: FunctionComponent<any & WrappedComponentProps> = ({
                       value={quantitySelected}
                       size="small"
                       type="number"
+                      min="1"
                       step={unitMultiplier}
                       onChange={(e: any) => {
-                        setState({
-                          ...state,
-                          quantitySelected: e.target.value,
-                        })
+                        if (e.target.value > 0){
```

Comment:
Could you please fix this Line error and rebase the CHANGELOG please?

Reply:
Also, I resolved the conflict of changelog! thanks for the warning!

Reply:
Thank you for update the changeling, it is the Line error from the PR check. You can see the detail from this link [https://github.com/vtex-apps/quickorder/runs/5012812236?check_suite_focus=true](url)

Reply:
Thanks @aurorahey ! I just changed the line and added the changed required! it was a space missing hahah